### PR TITLE
[VERIFYME] hardware/interfaces: Add thermal init NULL patch

### PR DESCRIPTION
**Pull request by ix5**
See https://github.com/sonyxperiadev/bug_tracker/issues/470#issuecomment-535689477
and https://android-review.googlesource.com/c/platform/hardware/interfaces/+/1320090

### CODE REVIEW

File: repo_update.sh [hunk 1]
```diff
@@ -114,6 +114,13 @@ apply_gerrit_cl_commit refs/changes/77/980177/3 0285b720ea752c8dcf28c35d794990e9
 apply_gerrit_cl_commit refs/changes/69/980169/2 a58def9e0ce610f1a349d5de31f267129a0a2397
 popd
 
+pushd $ANDROOT/hardware/qcom/audio
```
Comment (stefanhh0) [inline repo_update.sh:117]: Needs to be $ANDROOT/hardware/interfaces

**Comment by jerpelea** [comment]
we should add on the new HAL ASAP

File: repo_update.sh (hunk 1 above)
Comment (ix5) [reply repo_update.sh]: Thanks! Corrected

**Comment by ix5** [comment]
We already have the thermal HAL, in fact it's the thermal HAL that is throwing the issue ;)

The `thermal.default` module is not doing anything useful, it's just boilerplate https://android.googlesource.com/platform/hardware/libhardware/+/refs/tags/android-10.0.0_r35/modules/thermal/thermal.c

Once we overhaul the power HAL stuff, we can also couple it to a thermal uprev to `@2.0` and properly implement it.

**Comment by chris42** [comment]
Shouldn't this also be in the new android-10_legacy branch?

**Comment by ix5** [comment]
Feel free to PR it.

**Comment by chris42** [comment]
will do

**Pull request merged**